src/player.rs

```diff
@@ -144,9 +144,6 @@ pub(crate) fn move_player(
 		match collision {
 			None => {},
 			Some(_) => {
-				// Now as long as the player is standing on a tile, this will keep triggering
-				// This looks like an easy fix with state transitioning, we will see when that's implemented
-				// If not the solution is also simple: we kick the player out of the monster tile :)
```

Comment:
We will still actually need to handle this whenever we add the buttons and the ability to exit out of the battle screen and back into playing. We'll also need to separate StartPlaying state from Playing state so that when exiting Battle it doesn't call the game setup again. So this problem isn't necessarily gone yet.